Signal/src/Models/OWSMessagesBubblesSizeCalculator.m

```diff
@@ -106,8 +119,12 @@ - (CGSize)sizeForSystemMessage:(TSInteraction *)interaction
         return [cachedSize CGSizeValue];
     }
 
-    CGSize result =
-        [OWSSystemMessageCell cellSizeForInteraction:interaction collectionViewWidth:layout.collectionView.width];
+    if (!self.referenceSystemMessageCell) {
+        self.referenceSystemMessageCell = [OWSSystemMessageCell new];
```

Comment:
take it or leave it: Maybe a readonly lazily populated getter so we don't have to worry about this being nil, should we ever decide to use it elsewhere?